### PR TITLE
Fabric Operations Console

### PR DESCRIPTION
**Pull request by ptippett**
Hyperledger Fabric Operations Console code used in IBM's IBM Blockchain Platform Offering.

### CODE REVIEW

**Comment by denyeart** [comment]
Looks good to me and I'm happy to be a sponsor!

File: labs/fabric-operations-console.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+layout: default
+title: Fabric Operations Console
+parent: labs
+---
+# Lab Name
+Fabric Operations Console
+
+# Short Description
+Hyperledger Fabric Operations Console code used in IBM's IBM Blockchain Platform Offering.
```
Comment (lehors) [inline labs/fabric-operations-console.md:10]: I don't think the repetition is necessary.
```suggestion
Hyperledger Fabric Operations Console code used in the IBM Blockchain Platform Offering.
```

File: labs/fabric-operations-console.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+# Scope of Lab
+Hyperledger Fabric Operations Console used in IBM's IBM Blockchain Platform Offering.
```
Comment (lehors) [inline labs/fabric-operations-console.md:13]: Same here.
```suggestion
Hyperledger Fabric Operations Console used in the IBM Blockchain Platform Offering.
```

File: labs/fabric-operations-console.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+_Enter the Github IDs for the set of initial committers._
+- [Varad Ramamoorthy](https://github.com/varadgit)
+- [David Huffman](https://github.com/dshuffma-ibm)
+- [Paul Tippett](https://github.com/ptippett)
+
+# Sponsor
+_Provide the name of your sponsor. A sponsor is required and must be a maintainer of one of the Hyperledger projects, a TSC member, or a WG chair._
+- [Dave Enyeart](https://github.com/denyeart)  - Hyperledger TSC Member
+# Pre-existing repository
+_If you currently have a Github repository that you wish to transfer to the Hyperledger Labs organization, please provide a link here. N/A.
```
Comment (lehors) [inline labs/fabric-operations-console.md:41]:
```suggestion
N/A.
```

**Review by lehors — APPROVED** [review]
A few typos but LGTM.

**Review by tkuhrt — CHANGES_REQUESTED** [review]
Waiting for changes @lehors requested prior to approving.

File: labs/fabric-operations-console.md (hunk 1 above)
Comment (ptippett) [reply labs/fabric-operations-console.md]: done.  Thanks!

File: labs/fabric-operations-console.md (hunk 2 above)
Comment (ptippett) [reply labs/fabric-operations-console.md]: Done.  Thanks!

File: labs/fabric-operations-console.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+- Maintain complete control over identities, channels, and smart contracts.
+- Join Peers to Channels and view channel membership as well as individual transactions and channel details.
+- Register, view, delete, and re-enroll CA Users.
+- View Ordering cluster and node information as well as view and modify consortium and channel membership.
+- View and modify channel capabilites and ordering service parameters.
+- Install and Instantiate chaincode.  Supports both 1.x and 2.x Lifecycle.
+- View, Create, Import and Export Organizations and Identities.
+- Role Based Access Control in UI to tightly control which Console users can perform which operations.
+
+The console relies on [GRPC web](https://grpc.io/docs/platforms/web/) to allow GRPC based communication with Orderers and Peers via Node.js.  Managmeent of Certificate Authorities is done via REST API so does not reqiure a GRPC Web Instance.
```
[the remark this answers is not in the document]
Comment (ptippett) [reply labs/fabric-operations-console.md]: Done.  Thanks!

File: labs/fabric-operations-console.md (hunk 3 above)
Comment (ptippett) [reply labs/fabric-operations-console.md]: Done.  Thanks!

**Review by lehors — APPROVED** [review]
Thanks.

**Review by tkuhrt — APPROVED** [review]

**Comment by ryjones** [comment]
Merged, even though this looks like I closed without merging.

**Pull request closed without merging**